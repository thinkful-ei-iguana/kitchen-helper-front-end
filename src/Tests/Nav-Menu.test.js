import React from 'react';
import ReactDOM from 'react-dom';
import NavMenu from '../Components/Nav-Menu';
import { BrowserRouter } from 'react-router-dom';
import renderer from "react-test-renderer";

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter>
        <NavMenu />
    </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer.create(<BrowserRouter>
        <NavMenu />
    </BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});