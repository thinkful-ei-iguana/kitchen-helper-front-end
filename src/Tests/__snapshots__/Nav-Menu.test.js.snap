// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<header
  className="Nav-Header"
>
  <div>
    <a
      className="LandingPageLink"
      href="/"
      onClick={[Function]}
    >
      Landing Page Link
    </a>
  </div>
  <div
    className="desktopMenu"
  >
    <div
      className="Header__logged-out"
    >
      <a
        className="login"
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
      <p />
      <a
        className="create-account"
        href="/create-account"
        onClick={[Function]}
      >
         
        Register
      </a>
      <p />
    </div>
  </div>
</header>
`;
